WHITEPAPER.md

```diff
@@ -414,17 +417,19 @@ The end result is a scalable, pay-as-you-go network for decentralized live video
 | `T` | Segment length in seconds | 2 seconds |
 | `N` | Number of active transcoders | 144 |
 | `RoundLength` | Length of time between election of a new round of transcoders | 1 day |
-| `CyclesPerRound` | Number of times each Transcoder is expected to call Reward() during a Round. | 2 |
+| `InflationRate` | The current annual target inflation rate of LPT. | 15% |
 | `RateLockDeadline` | Transcoders rates lock in this amount of time prior to the next round start time so that delegators can review and delegate accordingly. | 6 hours |
 | `UnbondingPeriod` | Time between entering unbonding state, and ability to withdraw the funds. | 1 month |
-| `PersistenceLength` | The minimum period that a receipt of data persistence must be provided in the decentralized storage solution. | 6 hours |
+| `VerificationPeriod` | The deadline for verifying a job claim after submission of the job claim. This also serves as the minimum period that a receipt of data persistence must be provided in the decentralized storage solution. | 6 hours |
```

Comment:
Should we specify time in blocks instead of hours?

Reply:
Yes the protocol should definitely specify it in blocks. I used hours for the whitepaper just so it would be more human interpretable...and these are just examples, not recommended values. Maybe I'll add a note that specifies times will be given in blocks in the protocol.